packages/eyes/src/generate-snapshots/generate-snapshots.ts

```diff
@@ -0,0 +1,108 @@
+import * as path from 'path';
+const StylableWebpackPlugin = require('@stylable/webpack-plugin');
+const config = require(path.join(process.cwd(), './.autotools/webpack.config.js'));
+import * as webpack from 'webpack';
+import * as glob from 'glob';
+import {HTMLSnapshotPlugin} from '@stylable/webpack-extensions';
+import {createElement} from 'react';
+import {renderToStaticMarkup} from 'react-dom/server';
+import {generateFilteringLogic} from './filter-logic';
+import {generateMapping} from './generate-mapping';
+import Registry, {importMeta, getCompName, IComponentMetadata} from '@ui-autotools/registry';
+import {consoleLog} from '@ui-autotools/utils';
+import {dedent} from './dedent';
+import {parseSnapshotFilename} from './filename-utils';
+
+importMeta();
```

Comment:
Move these three lines to `generateSnapshots`. Since this is not a standalone script importing it shouldn't do anything.